nl.emphisia.icon.json

```diff
@@ -0,0 +1,52 @@
+{
+    "app-id" : "nl.emphisia.icon",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "master",
+    "sdk" : "org.gnome.Sdk",
+    "runtime-version" : "46",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "folder_icon",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "build-args" : [
+            "--share=network"
+        ],
+        "env" : {
+            "RUST_BACKTRACE" : "1",
+            "RUST_LOG" : "folder_icon=debug"
+        }
+    },
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "folder_icon",
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type": "archive",
+                    "url" : "https://github.com/youpie/Iconic/releases/latest/download/Iconic-Release.tar.xz",
+                    "sha256" : "https://github.com/youpie/Iconic/releases/latest/download/Iconic-Release.tar.xz.sha256sum"
```

Comment:
this is not a valid checksum. Where in the documentation does it says that that URL is valid? So that I can get it fixed.

Reply:
note that if you tried to build it, it should have failed.

Reply:
I have tried to build it using Gnome builder, but I think I am doing something very wrong them, as it builds normally there. Maybe I forgot to reload the configuration.
It does not say anywhere that a link is supported. but because I did not get any errors I assumed it did work :)
my apologies